### ISSUE TITLE
Menu dropdown should close on `Esc`.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#3396]
## Description

Based on discussion in slack thread:
https://gsa-tts.slack.com/archives/C3F14AHSQ/p1587058923090100

`Esc` key should be able to close the main menu and reset its subnavs.

Involves:
- [x] Updating `navigation.js`
- [x] Updating unit tests

## Steps to reproduce the issue

1. Visit [extended megamenu w/ dropdowns](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/release-2.6.0/components/preview/header--extended-mega.html)
2. Expand menu item
3. Hit `esc`, menu doesn't close

**Comment by mejiaj** [comment]
We should be able to do something like:
```JS
keydown: {
  [NAV_CONTROL]: keymap({ Escape: closeDropdown }),
}
```

Involves:
- [ ] Updating `navigation.js`
- [ ] Updating unit tests
